src/System.Net.Http/tests/FunctionalTests/ResponseStreamTest.cs

```diff
@@ -90,6 +91,54 @@ await client.GetAsync(HttpTestServers.RemoteEchoServer, HttpCompletionOption.Res
             }
         }
 
+        [ActiveIssue(6231, PlatformID.Windows)]
+        [Fact]
+        public async Task ReadAsStreamAsync_InvalidContentLengthResponse_ThrowsIOException()
+        {
+            var server = new TcpListener(IPAddress.Loopback, 0);
+            Task serverTask = ((Func<Task>)async delegate {
+                server.Start();
+                using (var client = await server.AcceptSocketAsync())
+                using (var stream = new NetworkStream(client))
+                using (var reader = new StreamReader(stream, Encoding.ASCII))
+                {
+                    // Read request
+                    string line;
+                    while (!string.IsNullOrEmpty(line = reader.ReadLine())) ;
+
+                    // Write response, with a potentially invalid content length
+                    using (var writer = new StreamWriter(stream, Encoding.ASCII))
+                    {
+                        string content = "This is some response content.";
+                        writer.Write("HTTP/1.1 200 OK\r\n");
+                        writer.Write($"Date: {DateTimeOffset.UtcNow:R}\r\n");
+                        writer.Write("Content-Type: text/plain\r\n");
+                        writer.Write($"Content-Length: {content.Length + 42}\r\n"); // incorrect length
```

Comment:
I didn't think we were able to use this construct yet?  The TFS mirrored builds are still C# 5, I think?